extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -613,6 +614,29 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
     return getCsiAmpAnalyticsConfig();
   }
 
+  /** @override */
+  letCreativeTriggerRenderStart() {
+    if (
+      this.element &&
+      this.element.parentElement &&
+      this.element.parentElement.tagName == 'AMP-STICKY-AD'
+    ) {
+      const stickyMsgListener = event => {
+        if (
+          getData(event) == 'fill_sticky' &&
+          event['source'] == this.iframe.contentWindow
+        ) {
+          this.renderStarted();
+          this.element.setAttribute('visible', '');
```

Comment:
One last request: we set iframe visibility to hidden here
https://github.com/ampproject/amphtml/blob/master/extensions/amp-ad/0.1/amp-ad-xorigin-iframe-handler.js#L278
Should also set iframe visibility instead of element visibility.

Reply:
done